torch/csrc/jit/constants.h

```diff
@@ -1,19 +1,23 @@
 #pragma once
-#include "ATen/ATen.h"
 #include "torch/csrc/jit/ivalue.h"
-#include "torch/csrc/jit/ir.h"
-#include "torch/csrc/jit/script/lexer.h"
+#include "torch/csrc/jit/source_range.h"
 #include "torch/csrc/WindowsTorchApiMacro.h"
 
 // helpers for handling constants in the IR
 // - create constant nodes from ints, floats, intlist, Tensors, and other types
 // - implement primitive constant ops.
 namespace torch { namespace jit {
 
+struct Graph;
+struct Value;
+
+// note: prefer g.insertConsant(val, loc) which does exactly the same thing
+// this function is only declared/defined here because its implementation is
+// closely related to the implementation of prim::Constant that is also in constants.cpp
```

Comment:
If this one is not really supposed to be used, then it would be good to e.g. add an underscore at the beginning of its name